src/App/Order/Model/PdkOrder.php

```diff
@@ -135,14 +132,6 @@ class PdkOrder extends Model
         'totalPriceAfterVat'    => 'int',
     ];
 
-    // TODO: v3.0.0 stop supporting deprecated attributes
-    protected $deprecated = [
```

Comment:
Ik zie geen andere changes die expliciet referenties hiernaartoe weghalen. Even ter dubbelcheck dat we niets gemist hebben :)

Reply:
De achterliggende functionaliteit kan behouden blijven voor de toekomst, misschien wil je voor de volgende major versies weer attributes deprecaten.

zie bijvoorbeeld https://github.com/myparcelnl/pdk/blob/cf5d191e8542c9f4e53f10a6a516605bfd531b1c/src/Base/Concern/HasAttributes.php#L516